src/main/java/com/github/skjolberg/packing/Space.java

```diff
@@ -2,77 +2,46 @@
 
 public class Space extends Dimension {
 
-	protected Space parent;
-	protected Space remainder;
+	private Space parent;
+	private Space remainder;
+
+	public int x; // width
```

Comment:
So what is wrong with setter and getter for x, y, z?

Reply:
nothing ... you prefer getters rather than direct access to fields ? we can put them back

Reply:
Okey, please put them back - I guess direct field access is ok for private inner classes, otherwise I never use them. There is no performance benefit really, because small methods like that are inlined by the JIT.

Reply:
sure, let's be consistent with the rest of the cofde !